package.json

```diff
@@ -22,27 +23,19 @@
   "author": "GitHub",
   "license": "MIT",
   "dependencies": {
-    "@actions/core": "^1.0.0",
-    "@actions/tool-cache": "^1.0.0",
+    "@actions/core": "^1.2.2",
+    "@actions/tool-cache": "^1.3.1",
     "semver": "^6.1.1"
   },
   "devDependencies": {
-    "@actions/io": "^1.0.0",
     "@types/jest": "^24.0.13",
     "@types/node": "^12.0.4",
     "@types/semver": "^6.0.0",
-    "husky": "^2.3.0",
+    "@zeit/ncc": "^0.21.0",
```

Comment:
NIT: consider a workflow step to make sure the dist/index.js file is up to date after running ncc. Its easy to overlook that file in PR's.